CMakeLists.txt

```diff
@@ -120,6 +120,7 @@ if (NOT PARTHENON_DISABLE_OPENMP)
     "If you want to build Parthenon without OpenMP, please rerun CMake with -DPARTHENON_DISABLE_OPENMP=ON")
   endif()
   set(ENABLE_OPENMP ON)
+  set(Kokkos_ENABLE_OPENMP ON)
```

Comment:
Could you make this a cache variable instead so that it doesn't override the user setting if the user explicitly disables it?

Reply:
Isn't that redundant with ```PARTHENON_DISABLE_OPENMP```, I could always warn the user to use ```PARTHENON_DISABLE_OPENMP``` instead if they are trying to alter `Kokkos_ENABLE_OPENMP` and tell them to use the parthenon equivalent.

Reply:
ping @AndrewGaspar

Reply:
I want it to be redundant - you might be in a situation where Kokkos + OpenMP is on the fritz for some reason, but still want to enable Parthenon's raw OpenMP. Making it a cache variable gives you an out so that you can disable Kokkos' OpenMP without disabling Parthenon's OpenMP.